includes/blocks/class-sensei-course-completed-actions-block.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * File containing the Sensei_Course_Completed_Actions_Block class.
+ *
+ * @package sensei
+ * @since 3.13.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Sensei_Course_Completed_Actions_Block
+ */
+class Sensei_Course_Completed_Actions_Block {
+
+	/**
+	 * Sensei_Course_Completed_Actions_Block constructor.
+	 */
+	public function __construct() {
+		$this->register_block();
+	}
+
+	/**
+	 * Register the block.
+	 *
+	 * @access private
+	 */
+	public function register_block() {
+		Sensei_Blocks::register_sensei_block(
+			'sensei-lms/course-completed-actions',
+			[
+				'render_callback' => [ $this, 'render_block' ],
+			],
+			Sensei()->assets->src_path( 'blocks/course-completed-actions' )
+		);
+	}
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array  $attributes Block attributes.
+	 * @param string $content    Block HTML.
+	 *
+	 * @return string Block HTML.
+	 */
+	public function render_block( $attributes, $content ): string {
+		$dom = new DomDocument();
+		$dom->loadHTML( $content );
+		$parent_div = $dom->getElementsByTagName( 'div' )->length > 0 ? $dom->getElementsByTagName( 'div' )[0] : '';
+		$anchor     = $parent_div && $parent_div->getElementsByTagName( 'a' )->length > 0 ? $parent_div->getElementsByTagName( 'a' )[0] : '';
```

Comment:
I'm receiving this error here:

> DOMDocument::loadHTML(): Empty string supplied as input in /Users/renatho/Workspace/automattic/Sensei/sensei/includes/blocks/class-sensei-course-completed-actions-block.php on line 50

But I'm wondering if the user decides to add another button before the "Find More Courses". A suggestion would be to add a fix anchor in that button with the attributes, like `href="#find-more-courses"`, and just replace it. WDYT? IIRC, Giannis used this approach in some place.

Reply:
I switched it to use the CSS class instead. Someone could easily change that link, but updating the CSS class is a bit more hidden. https://github.com/Automattic/sensei/pull/4266/commits/e044acf3ee821691bdac847fa4225b1b4d576d49